### DISCUSSION TITLE
Popup opacity

### DISCUSSION THREAD

**Discussion by tomkoid** [hyprwm/Hyprland#2013] [Q&A] [answered]
Is there a way to set opacity to 1.0 for all popups? All of my popups are currently transparent and some applications just don't work with it right.

![image](https://user-images.githubusercontent.com/67477750/230876589-58bdc087-6708-489a-9706-53e691b4ad2c.png)

**Comment by vaxerski** [comment, marked as answer]
that looks like a theme issue

**Reply by tomkoid** [reply]
hmmm I changed the theme and now it's normal but I can't replicate this issue in any other window manager/desktop environment with the exact same theme.
